JhipsterXamarin/JhipsterXamarin/JhipsterXamarin/Logs.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using MvvmCross;
+using MvvmCross.Logging;
+
+namespace JhipsterXamarin
+{
+    public static class Logs
+    {
+        public static IMvxLog Instance { get; } = Mvx.IoCProvider.Resolve<IMvxLogProvider>().GetLogFor("JhipsterXamarin");
```

Comment:
prefer the use of injection like this 

```
    private readonly IMvxLog _log;
    public MyViewModel(IMvxLogProvider logProvider)
    {
        _log = logProvider.GetLogFor<MyViewModel>();
    }
```

Reply:
How do we log outside of VMs ?

Reply:
the logger is not link to the VM you can use it like this   
```
public MyService(IMvxLogProvider logProvider)
    {
        _log = logProvider.GetLogFor<MyService>();
    }
```

Reply:
How to do it inside the main app ?

Reply:
you have to instantiate it but just for App and inject for all other use cases

Reply:
Tried to do it in 3634f4b